public/pom.xml

```diff
@@ -29,6 +29,9 @@
 		<module>transactions-jndi-provider</module>
 		<module>atomikos-transactions-essentials</module>
 		<module>transactions-remoting</module>
+		<module>transactions-spring-boot</module>
+		<module>transactions-spring-boot-starter</module>
```

Comment:
So, what is the relationship between this PR and https://www.atomikos.com/Blog/TransactionsEssentials5dot0dot9#Feature188756_Add_spring_boot_starter ?